Library/Homebrew/cmd/audit.rb

```diff
@@ -676,6 +676,10 @@ def audit_text
     if text =~ /def plist/ && text !~ /plist_options/
       problem "Please set plist_options when using a formula-defined plist."
     end
+
+    if text =~ %r{require "language/go"} && text !~ /go_resource/
```

Comment:
Feel free to ignore this, but I'm feeling like we shouldn't be using regular expressions where they aren't needed, even if that makes for slightly less uniform code. In this case I'd be advocating for `String#include?`. One issue is that it's easy to forget to escape some special characters in regular expressions (not the case here), another that (in theory) a simple substring search should be significantly cheaper.